Doc/Makefile

```diff
@@ -40,7 +40,19 @@ help:
 
 build:
 	-mkdir -p build
-	$(BLURB) merge -f build/NEWS
+	# Look first for a Misc/NEWS file (building from a source release tarball
+	# or old repo) and use that, otherwise look for a Misc/NEWS.d directory
+	# (building from a newer repo) and use blurb to generate the NEWS file.
+	@if [ -f  ../Misc/NEWS ] ; then \
+		echo "Using existing Misc/NEWS file"; \
+		cp ../Misc/NEWS build/NEWS; \
```

Comment:
I don't think so. $INSTALL_DATA is not currently used in the Doc Makefile and we're not installing anything.  The cp is just for a temporary build file.